lib/libpaper.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="https://apps.0install.net/lib/libpaper.xml" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>libpaper</name>
+  <summary xml:lang="en">LibPaper: library and tools for handling papersize</summary>
+  <description xml:lang="en">The paper library and accompanying files are intended to provide a simple way for applications to take actions based on a system- or user-specified paper size. It lets program automatically recognize a lot of different papers with their properties (actually their size). It also includes a program for accessing paper information from shell scripts. 
+
+This release is quite minimal, its purpose being to provide really basic functions (obtaining the system paper name and getting the height and width of a given kond of paper) that applications can immediately integrate. 
+</description>
+  <icon href="https://raw.githubusercontent.com/0install/apps.0install.net/master/lib/libpaper.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/apps.0install.net/master/lib/libpaper.png" type="image/png"/>
```

Comment:
Could you fix these URLs?
https://raw.githubusercontent.com/0install/apps/master/lib/libpaper.ico
https://raw.githubusercontent.com/0install/apps/master/lib/libpaper.ico

Reply:
fixed